Lib/json/tool.py

```diff
@@ -16,6 +16,18 @@
 import sys
 
 
+def parse_indent(indent):
+    """Parse the argparse indent argument."""
+    if indent == 'None':
+        return None
+    if indent == r'\t':
+        return '\t'
```

Comment:
Cross-referencing [bpo-29636](http://bugs.python.org/issue29636).

@serhiy-storchaka so do you think it's best to simplify so you must either pass `--indent` an int or `None`. Anything that cannot be converted to an int besides `None` would throw an error?